build.sh

```diff
@@ -72,7 +72,7 @@ function build_fluentbit() {
   cd submodules/fluent-bit
   mkdir -p build
   cd build
-  cmake .. -DCMAKE_INSTALL_PREFIX=$subagentdir/fluent-bit -DFLB_HTTP_SERVER=On
+  cmake .. -DCMAKE_INSTALL_PREFIX=$subagentdir/fluent-bit -DFLB_HTTP_SERVER=On -DCMAKE_BUILD_TYPE=RELWITHDEBINFO
```

Comment:
Do you need to do the same thing in Dockerfile.windows?

Reply:
I imagine so, as it looks like the windows build doesn't have debug symbols. I am not familiar with the windows build, do you want me to add that change? (RELEASE -> RELWITHDEBINFO)

Reply:
Feel free to make that change... It still uses cmake so hopefully the syntax is the same?

Reply:
Done!